src/Common/Commands.Common.Network/Version2017_10_01/Models/ExpressRouteCircuitPeeringType.cs

```diff
@@ -12,9 +12,9 @@ namespace Microsoft.Azure.Management.Internal.Network.Version2017_10_01.Models
 {
 
     /// <summary>
-    /// Defines values for ExpressRouteCircuitPeeringType.
+    /// Defines values for ExpressRoutePeeringType.
     /// </summary>
-    public static class ExpressRouteCircuitPeeringType
+    public static class ExpressRoutePeeringType
```

Comment:
You can't rename this class. Please change the name back. If you are having issues with your new classes, either use fully qualified names or alias usings.

Reply:
This was renamed in swagger

Reply:
If the swagger was changed, that would be a new version. You can't just change the `Version2017_10_01` models to match the current swagger. I'm assuming there were other changes beyond renaming this one class to the swagger from that point in time. This is in the common Network project, so it is used by other non-Network modules. Unfortunately, I'm not sure of the process for generating new common versions. @praries880 Any insight?

Reply:
Sorry....did not realize this is 2017_10_01. Fixed